src/code42cli/cmds/alerts.py

```diff
@@ -159,14 +164,33 @@ def clear_checkpoint(state, checkpoint_name):
     "--or-query", is_flag=True, cls=searchopt.AdvancedQueryAndSavedSearchIncompatible
 )
 @opt.sdk_options()
+@click.option(
+    "--include-all",
+    default=False,
+    is_flag=True,
+    help="Display complete list of data items from the first level of the nested response.",
```

Comment:
Is `properties` a better word for `data items` ?

Reply:
Actually, see this comment: https://github.com/code42/code42cli/pull/133/files#r470628459